Marlin/src/lcd/marlinui.cpp

```diff
@@ -1035,9 +1035,13 @@ void MarlinUI::init() {
         uint8_t abs_diff = ABS(encoderDiff);
 
         #if ENCODER_PULSES_PER_STEP > 1
-          static int8_t lastEncoderDiff;
-          TERN_(HAS_TOUCH_SLEEP, if (lastEncoderDiff != encoderDiff) wakeup_screen());
-          lastEncoderDiff = encoderDiff;
+          #if HAS_TOUCH_SLEEP
+            static int8_t lastEncoderDiff;
+            if (lastEncoderDiff != encoderDiff) {
+              wakeup_screen();
+              lastEncoderDiff = encoderDiff;
+            }
+          #endif
         #endif
```

Comment:
actually would this change be better?

```diff
-        #if ENCODER_PULSES_PER_STEP > 1
-          #if HAS_TOUCH_SLEEP
             static int8_t lastEncoderDiff;
             if (lastEncoderDiff != encoderDiff) {
               wakeup_screen();
               lastEncoderDiff = encoderDiff;
             }
-          #endif
-        #endif

+        #if (ENCODER_PULSES_PER_STEP > 1) && HAS_TOUCH_SLEEP
           static int8_t lastEncoderDiff;
           if (lastEncoderDiff != encoderDiff) {
             wakeup_screen();
             lastEncoderDiff = encoderDiff;
           }
+        #endif
```

Reply:
Looks good to me.  I am not real well versed in GIT, but Looks like you already made these changes so I don't have to do anything.  LMK if you need something from me.

Reply:
well I didn't actually make the changes, this is just a suggestion. you or a member has the authority to actual commit the change.